packages/core-components/package.json

```diff
@@ -69,6 +69,7 @@
     "@types/react-sparklines": "^1.7.0",
     "ansi-regex": "^6.0.1",
     "classnames": "^2.2.6",
+    "clsx": "^2.1.1",
```

Comment:
We can stick to using `classnames` rather than adding this right?

Reply:
Yes, removed.